dynatrace support for open telemetry (#635)
cap-js/docs: java/observability.md

[Open Telemetry](https://opentelemetry.io/) is an Open Source framework for observability in cloud applications. Applications can collect signals (distributed traces and metrics) and send them to observability front ends that offer a wide set of capabilities to analyze the current state or failures of an application. On the Business Technology Platform, for example, the  [SAP BTP Cloud Logging service](https://help.sap.com/docs/cloud-logging) is offered as a front end for these purposes.

> CAP Java applications can easily be configured to connect to SAP BTP Cloud Logging Service or Dynatrace. By configuring Open Telemetry support with one of these service in a CAP Java application, the application automatically benefits from the following features:

Reviewer: I'm not sure if I understand this. Could you rephrase it? Here's my understanding:
```suggestion
CAP Java applications can easily be configured to connect to SAP BTP Cloud Logging Service or Dynatrace. In your CAP Java application, you configure one of these services inside the Open Telemetry configuration. Then the application automatically benefits from the following features:
```
